privacyidea/lib/tokens/webauthntoken.py

```diff
@@ -0,0 +1,1195 @@
+                                  "space-separated list of AAGUIDs. Per default all authenticators are acceptable."),
+                        'group': GROUP.TOKEN
+                    },
+                    WEBAUTHNACTION.USER_VERIFICATION_REQUIREMENT: {
+                        'type': 'str',
+                        'desc': _("Whether the user's identity should be verified when rolling out a new WebAuthn "
+                                  "token. Default: preferred (verify the user if supported by the token)"),
+                        'group': GROUP.TOKEN,
+                        'value':    [
+                            "required",
+                            "preferred",
+                            "discouraged"
+                        ]
+                    },
+                    WEBAUTHNACTION.PUBLIC_KEY_CREDENTIAL_ALGORITHM_PREFERENCE: {
+                        'type': 'str',
+                        'desc': _("Which algorithm to use for creating public key credentials for WebAuthn tokens"
+                                  "Default: ecdsa_preferred"),
+                        'group': GROUP.TOKEN,
+                        'value': [
+                            "ecdsa_preferred",
+                            "ecdsa_only",
+                            "rsassa-pss_preferred",
+                            "rsassa-pss_only"
+                        ]
+                    },
+                    WEBAUTHNACTION.AUTHENTICATOR_ATTESTATION_FORM: {
+                        'type': 'str',
+                        'desc': _("Whether to request attestation data when enrolling a new WebAuthn token."
+                                  "Note: for u2f_req to work with WebAuthn, this cannot be set to none. "
+                                  "Default: direct (ask for non-anonymized attestation data)"),
+                        'group': GROUP.TOKEN,
+                        'value': [
+                            "none",
+                            "indirect",
+                            "direct"
+                        ]
+                    },
+                    WEBAUTHNACTION.AUTHENTICATOR_ATTESTATION_LEVEL: {
+                        'type': 'str',
+                        'desc': _("Whether and how strictly to check authenticator attestation data."
+                                  "Note: If the attestation form is none, the attestation level needs to also be none."
+                                  "Default: untrusted (attestation is required, but can be unknown or self-signed)"),
+                        'group': GROUP.TOKEN,
+                        'value': [
+                            "none",
+                            "untrusted",
+                            "trusted"
+                        ]
+                    },
+                    WEBAUTHNACTION.REQ: {
+                        'type': 'str',
+                        'desc': _("Only the specified WebAuthn-tokens are allowed to be registered."),
```

Comment:
What is the admin supposed to enter here?
Can you give a further hint?

Reply:
This is the same as for U2F. It's a quite complex field, whose usage is described in detail in the documentation. Not sure how to better describe it in such a confined space.

Reply:
OK, the same as in u2f
https://privacyidea.readthedocs.io/en/latest/policies/enrollment.html#u2f-req